### ISSUE TITLE
a new dev channel?

### ISSUE THREAD

**Issue by Quezler** [Anuken/CoreBot#12]
Currently the discord has a bunch of discussion channels for normal players, a bunch for modding, but only one for development, which is technically only meant for discussing mindustry related development.

But in practice its more of an offtopic/social/showcase/suggestion/etc channel inhabited by developers and those curious as to what they're up to.

My opinion is that it may be best if developers get a secondary channel thats still related to mindustry's development, but a bit more informal, where people ~~can~~ are allowed to suggest stuff for the java programmers in there, where they can share their ideas and random pulls, etc, keeping the actual `#development` channel a bit more official and true to its purpose. 
(people complaining about offtopic but still development-ish related messages in there is way to common)

While we technically could post in `#screenshots/social/designs/spriting/multiplayer/etc` its a bit hard to keep track of what is where, and it'll be harder to read up on unread messages if there is a lot of normal user content in there.

For this purpose i've kinda been brainstorming all day (was traveling, nothing else to do) with possible suggestions and/or inspiration as to what a potential channel name could be. (there's weird stuff in there, but hey, its brainstorming, that just happens)

The view described above is entirely my own, feel free to implement or disregard it. 🙂 

**TLDR** adding a discord channel for those involved with the java side of mindustry?
("_all-in-one channel for those who operate on a level above json modding_")

```Devrant
Developers anonymous
Ooftopic
Foobar
Git together
~~social~~ introverts
Devbay
Devhub
Rabbithole
miscdev
Showcase
Feathub
#none-of-you-are-helping 
#your-making-it-worse
#please-leave
#go-away
Do you know da (dev) wey?
Technical
Tech
Science
Scholarly
Developing
(En)tsngled
Mechanical
Industrial
Ponder
Pond
aristocracy
Elite
Upper class
Nobility
Contributor
Contributors
Contributions
Powerhouse
Aperture science
Speedy idea goes in, speady code goes out
Devportal
Warpgate
Politics
Dominion
Bundles
Cackle
Chicken coop
Dev coop
Eggs
Cauldron
Advancement
Peasants begone
Laborers
Hazbin
Aperture
Binary
Morse
E=MC2
Roombotics
Invention suspension
Isle of ~~dog~~ devs
Borough
Forks
Forking
It's raining ~~meatballs~~ suggestions
Coding
Darkroom
Adventofcode
Engineers
Architects
Inventors
Schemers
Innovators
Pioneers
Coiner
Mintery
mint
Coining
Masterminds
Sages
Think-tank
Napoleon branch aparte
Julius racecar
Duopatra
Kitty litter
IRC
Frontdesk
Workspace
Davenport
Devport
Counter
Ding dong
Beaker
Ohno
Lurker
Sandbox
Lavatory 
Galactic senate 
Library
Librarians
Bookshelf 
Scholars
Bookworms
Intellectuals 
Tutors
Comrades
Conference
Club penguin 
Tech support
Rally
Incubator
Hideaway
Sanctuary
Stronghold
Shelter
Oh no tre damme
Sanctum sanctorous
Cat scratch pole
Mausoleum
Food fight
Tomb
Black market 
Trench
Cesspool
Parliment
Snek
Last resort
Mercenaries 
Bridge of death 
Babysitters
Wardens
Daycare
Shepherds
Preservers 
Egg basket 
Custodians
Ministry of silly ~~walks~~ ideas
The royal society for putting things on top of other things 
Don't mention the war
Your offtopic is in another castle
Council
Congress
Municipal dome
Conclave
Twilight council
Daelaam
Muggles
Hivemind
Courtroom
Bulletins
Senpai notice me
High knee council
Enter the ~~florpus~~ offtopic
Demilitarised zone
Basement 
Cellar
Hive
Swarm
Outpost 
Stronghold
Zone
Commonwealth
Privileged
Forks
Pulls
Little bobby router 
Litter
Attention seekers
Feed me
You must return when the moon is full and the fox is borrowed
Untrusted
Pillowfight
African & European swallows 
Your mother was a router and your father smelled of elderberries
Those-who-use-gradlew
Bodgewartz
Your ethernal reward
Yellow sub machine
Candlemaker
Pootis
Additional pylons
Slithering coresnek
```
🥔

**Comment by DeltaNedas** [comment]
why not just  #devdiscussion
also remove user content thanks :+1:

**Comment by Quezler** [comment]
that channel name doesn't look pleasing to the eye ^

also, what exactly do you mean with that last part?

**Comment by DeltaNedas** [comment]
schematics is just pixel art
mods is ~30% me bumping vbucks mod after big updates
maybe keep maps and plugins but theyre on thin ice

**Comment by Quezler** [comment]
closing this since new development channels have been created on the discord
